src/core/qgsvectorlayer.h

```diff
@@ -406,12 +406,12 @@ class CORE_EXPORT QgsVectorLayer : public QgsMapLayer, public QgsExpressionConte
         , readExtentFromXml( readExtentFromXml )
       {}
 
-      //! Sets to true if the default layer style should be loaded
+      //! Sets to VAL_TRUE if the default layer style should be loaded
```

Comment:
Unrelated suggestion

```suggestion
      //! Set to VAL_TRUE if the default layer style should be loaded
```